www/perl-lib/SMT/Utils.pm

```diff
@@ -64,6 +64,8 @@ use constant MYSQL_TEXT_TYPE_SIZE => 65535;
 
 use constant PATCH_SUMMARY_SIZE_CHARS => 512;
 
+use constant SMT_USER_AGENT => 'smt/3.0.46 ' . WWW::Curl::Easy->version();
```

Comment:
do you think we really need the curl versions?

Reply:
It wouldn't hurt, also, since this will trigger a new release, should I bump the version in the user-agent string first?

Reply:
After a bit of back & forth, it's there :D I'll make a small test locally to ensure it's hooked properly